src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -76,4 +86,22 @@ object Lorre extends App with LazyLogging {
   }
 
 
+  def processTezosAverageFees(): Try[Unit] = {
+    logger.info("Processing latest Tezos fee data...")
+    val operationKinds = List("seed_nonce_revelation", "delegation", "transaction", "activate_account", "origination", "reveal", "double_endorsement_evidence", "endorsement")
+    val fees = operationKinds.map{ kind =>
+      ApiOperations.averageFee(kind)
```

Comment:
`ApiOperations` contains functions for fetching data dynamically from the database. This function is now used for writing data to the database. Let's move this also to `TezosDatabaseOperations`.

Reply:
This is incorrect, averageFee does read from the database, and belongs in APIOperations

Reply:
"Dynamically" is key here. ApiOperations responds to API calls and pulls data from precooked tables. Therefore, it is the right place to put the function which reads the fees from the existing table. However, the database static read and write logic should reside elsewhere.

Reply:
Okay, got it, moved the function.